ert_shared/plugins/__init__.py

```diff
@@ -1 +1,13 @@
 from .plugin_manager import ErtPluginContext, ErtPluginManager
+from .visualization_plugin_handler import VisualizationPluginHandler
+
+
+def launch_visualization_plugin(args):
+    pm = ErtPluginManager()
+    handler = VisualizationPluginHandler()
+    pm.hook.register_visualization_plugin(handler=handler)
+    try:
+        vis_plugin = handler.get_plugin(args.name)
+        vis_plugin.run()
+    except UserWarning as e:
+        raise SystemExit(str(e))
```

Comment:
Maybe just have `get_plugin` return `None` in the case of no matching plugin name, and then run/exit based on that?

Reply:
I think the plugin handler should blow up early if you try to use a plugin that is not installed :shrug:

Reply:
Alright! But let's choose something else than UserWarning